lib/ddtrace/configuration/settings.rb

```diff
@@ -161,6 +161,7 @@ def logger=(logger)
         end
 
         settings :advanced do
+          # This should never be reduced, as it can cause the resulting profiles to become biased
           option :max_events, default: 32768
```

Comment:
Given how strongly written this comment is: how did we get to 32768 as THE minimum value to avoid bias?

Reply:
I wasn't the one who picked the value, but it seems reasonable:

* We sample at most 100 times per second, so this would be enough to sample 5 threads for the 60 second period even if we were sampling at full speed.
* In reality we're probably going to be closer to [sampling 30-ish](https://github.com/DataDog/dd-trace-rb/pull/1695#issuecomment-926497932) times per second, which would fit within the 16-thread-per-sample target for this PR.

I have an internal ticket (PROF-4072) to improve how we drop events so to not induce bias.